pyproject.toml

```
@@ -0,0 +1,230 @@
+[project]
+name = "dea-tools"
+dynamic = ["version"]
+description = "Functions and algorithms for analysing Digital Earth Australia data."
+authors = [
+    { name = "Geoscience Australia", email = "earth.observation@ga.gov.au" }
+]
+readme = "README_tools.md"
+license = "Apache-2.0"
+keywords = [
+    "earth observation",
+    "satellite data",  
+    "geospatial analysis",
+    "remote sensing",
+    "digital earth australia",
+    "datacube",
+]
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Intended Audience :: Science/Research",
+    "Topic :: Scientific/Engineering :: GIS",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+]
+requires-python = ">=3.10,<4.0"
+dependencies = [
+    "aiohttp>=3.5.0",
+    # "ciso8601>=2.2.0",  # blocks Python 3.13 installation on Windows
+    "dask>=2023.1.0",
+    "dask-ml>=2021.10.17",
+    # "eo-tides>=0.7.4",  # requires notebook updates
+    "fiona>=1.10.0",
+    "folium>=0.16.0",
+    "geopandas>=0.10.0",
+    "geopy>=2.0.0",
+    "imageio>=2.20.0",
+    "joblib>=1.0.0",
+    "lxml>=5.0.0",
+    "matplotlib>=3.8.0",
+    "numpy>=1.26.0",
+    "odc-algo>=1.0.0",
+    "odc-geo[warp,tiff]>=0.4.7",
+    "OWSLib>=0.26.0",
+    "packaging>=22.0",
+    "pandas>=2.2.0",
+    "pyproj>=3.7.0",
+    "pyTMD==2.1.6",
+    "pytz>=2022.1",
+    "rasterio>=1.3.11",
+    "rasterstats>=0.16.0",
+    "requests>=2.25.0",
+    "rioxarray>=0.10.0",
+    "seaborn>=0.10.0",
+    "scikit-image>=0.22.0",
+    "scikit-learn>=1.4.0",
+    "scipy>=1.14.1",
+    "shapely>=2.0.5",
+    "traitlets>=5.0.0",
+    "tqdm>=4.55.0",
+    "xarray>=2022.3.0",
+]
+
+[project.urls]
+Homepage = "https://github.com/GeoscienceAustralia/dea-notebooks"
+Repository = "https://github.com/GeoscienceAustralia/dea-notebooks"
+
+[project.optional-dependencies]
+all = ["dea-tools[datacube,stac,jupyter,dask_gateway,hdstats,notebooks]"]
+datacube = [
+    "datacube>=1.8.5", 
+    "odc-ui>=0.2.1",
+    # "datacube-ows>=1.8.30",  # has unwanted psycopg2 dependency
+]
+stac = [
+    "odc-stac>=0.3.0",
```

```suggestion
    "odc-stac[botocore]>=0.3.0",
```

Suggest including botocore, since when using dea-tools, we might expect people to connect to DEA's AWS bucket. If botocore is not included, users will receive an error message saying they need to install it.

---

Great idea!